pkg/webhook/stats_reporter.go

```diff
@@ -42,6 +43,7 @@ func init() {
 // StatsReporter reports webhook metrics.
 type StatsReporter interface {
 	ReportValidationRequest(ctx context.Context, response requestResponse, d time.Duration) error
+	ReportValidationRequestAndType(ctx context.Context, response requestResponse, dryRun *bool, d time.Duration) error
```

Comment:
nit: to be consistent

```suggestion
	ReportValidationRequestAndType(ctx context.Context, response requestResponse, isDryRun *bool, d time.Duration) error
```